### PR TITLE
feat(ct): DeployDelayedWETH script

### PR DESCRIPTION
**Pull request by smartcontracts**
Adds a new deployment script for deploying a new DelayedWETH proxy contract.

### CODE REVIEW

File: packages/contracts-bedrock/scripts/DeployDelayedWETH.sol [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+            revert("DeployDelayedWETH: unknown selector");
+        }
+    }
+
+    function set(bytes4 _sel, string memory _value) public {
+        if (_sel == this.release.selector) {
+            require(LibString.eq(_value, ""), "DeployDelayedWETH: release cannot be empty");
+            _release = _value;
+        } else if (_sel == this.standardVersionsToml.selector) {
+            require(LibString.eq(_value, ""), "DeployDelayedWETH: standardVersionsToml cannot be empty");
```
Comment (tynes) [inline packages/contracts-bedrock/scripts/DeployDelayedWETH.sol]: looks like you forgot a `!` here

Comment (smartcontracts) [reply packages/contracts-bedrock/scripts/DeployDelayedWETH.sol]: fastest review of all time

Comment (smartcontracts) [reply packages/contracts-bedrock/scripts/DeployDelayedWETH.sol]: fixed

**Comment by mslipper** [comment]
This looks good, it's just missing a test for the new OPCM method. You can add a test using a similar pattern to the ones in `apply_test.go`.

**Comment by smartcontracts** [comment]
@mslipper I don't see where the other bootstrap method is being tested, can you point me to it?

**Review by mslipper — APPROVED** [review]

**Pull request merged**